website/site/content/docs/configuration-options.md

```diff
@@ -168,3 +169,16 @@ fields:
   - {label: "Body", name: "body", widget: "markdown"}
 ```
 
+### `editor`
```

Comment:
For such a simple setting, it seems odd to require a "more details below" section. The only complication to it is the fact that it's a nested setting, which is also odd. I suppose the reasoning is that there might be more editor settings in the future. In that case, I think it makes more sense to format the section like the other settings that have sub-settings. Something along the lines of:

```
### `editor`

This setting changes options for the editor view of the collection. It has one option so far:

- `preview`: set to `false` to disable the preview pane for this collection; defaults to `true`

[continue with your example as coded, but remove the name and label lines]
```

Reply:
Done with the exception of the "defaults to" based on @erquhart's note about it not changing a default.

Reply:
I think what @erquhart meant by that was that the original wording sounded like it might mean the setting effectively toggled the editor's preview button to "hide" by default, but left the option for the user to unhide it by toggling the button. In other words, a different meaning of "default."

On the other hand, naming the default value for the setting itself follows the pattern used in all other settings, and I don't think causes the confusion above.

Reply:
And sorry to sound nit-picky! I really do appreciate that you've identified the need for this in the docs and added it.